plone/app/upgrade/v50/final.py

```diff
@@ -92,6 +95,27 @@ def to502(context):
     loadMigrationProfile(context, 'profile-plone.app.querystring:upgrade_to_5')
 
 
+def _fix_typo_in_toolbar_less_variable(context):
+    registry = getUtility(IRegistry)
+    rreg = registry.forInterface(
+        IResourceRegistriesSettings,
+        prefix='plone'
+    )
+    _marker = list()
+    value = rreg.lessvariables.get('plone-toolbar-font-secundary', _marker)
+    if (value == _marker):
+        return
+    if 'plone-toolbar-font-secondary' in rreg.lessvariables:
+        logger.warn(
+            'Try to migrate registry value "plone-toolbar-font-secundary" to '
+            '"plone-toolbar-font-secundary", but latter already exists. '
+            'Migration to fix the typo is not executed.'
+        )
+        rreg.lessvariables['plone-toolbar-font-secondary'] = value
+        del rreg.lessvariables['plone-toolbar-font-secundary']
```

Comment:
The indentation looks wrong. Bit hard to see with my comment in between. :-)
It is now:

```
if secondary without typo already exists:
    Warn that migration is not executed.
    Execute migration anyway.
```